### PR TITLE
LPD-31394 Remove LPS-165481 FF

### PR DESCRIPTION
**Pull request by GaborKomaromi**
https://liferay.atlassian.net/browse/LPD-31394

Removed FF to avoid error  'Feature flag LPS-165481 is not available for company 0' is showed in logs

### CODE REVIEW

**Comment by GaborKomaromi** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase [hunk 1]
```diff
@@ -1727,7 +1724,7 @@ definition {
 	@description = "This is a use case for LPS-190015, TC1. Publishing a single webcontent by multi publish option."
 	@priority = 3
 	test PublishOneWebContentWithMultiPublishOption {
-		property custom.properties = "feature.flag.LPS-165481=true";
+		property test.liferay.virtual.instance = "false";
```
Comment (4lejandrito) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:1727]: Hey @GaborKomaromi why is `test.liferay.virtual.instance = "false"` necessary now?

**Comment by GaborKomaromi** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase (hunk 1 above)
Comment (GaborKomaromi) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase]: Deleted them, left them as placeholder as SF was failing due to a line break

**Comment by GaborKomaromi** [comment]
ci:test:relevant

**Review by 4lejandrito — APPROVED** [review]

**Comment by GaborKomaromi** [comment]
ci:forward

**Review by vendeltoreki — APPROVED** [review]

**Comment by GaborKomaromi** [comment]
see: https://github.com/brianchandotcom/liferay-portal/pull/152494

**Pull request closed without merging**